proxy/src/main/java/org/apache/accumulo/proxy/ProxyServer.java

```diff
@@ -189,31 +188,23 @@ public void onRemoval(RemovalNotification<UUID,ConditionalWriter> notification)
 
   private final ThriftServerType serverType;
 
-  public ProxyServer(Properties props) {
+  public ProxyServer(Properties proxyProps, Properties clientProps) {
```

Comment:
I think combining these two sets of properties maintains a better abstraction between the implementation of the server and the organization of the parameters required to start the server.

Another option would be to pass in the proxy server properties, along with a Connector that has already been constructed.

Reply:
They are passed in separately because they are read from two different files.  I can look into combining them but it might more confusing.

Reply:
I don't think ProxyServer's implementation needs to know that the caller read those properties from multiple sources.